frontend/common/src/helpers/nameUtils.tsx

```diff
@@ -76,3 +77,74 @@ export const getFullNameHtml = (
   }
   return `${firstName} ${lastName}`;
 };
+
+/**
+ * Wraps common abbreviations in abbr tags to make them more accessible
+ *
+ * @param text   text to wrap
+ * @param intl   react-intl object
+ * @param title  abbreviation title
+ */
+export const wrapAbbr = (
+  text: React.ReactNode,
+  intl: IntlShape,
+  title?: string,
+): JSX.Element => {
+  const stringifyText = text && text.toString(); // grabs text from React.ReactNode (is there a better way to get text from React.ReactNode type?)
+  if (typeof stringifyText !== "string") {
```

Comment:
Could you use the logger in frontend/common/src/hooks/useLogger.ts instead of console?  This will allow us to log other places, like Azure AppInsights, for example.

Reply:
Yeah, that works OK.  If you do need to use the logger outside a component you can import the logger directly instead of using the hook, like in frontend/common/src/components/Auth/AuthenticationContainer.tsx .